packages/tokens/scripts/build.ts

```diff
@@ -85,6 +85,12 @@ StyleDictionaryPackage.registerTransformGroup({
     transforms: ['attribute/cti', 'name/cti/kebab', 'font-size/rem', 'size/px', 'color/css', 'color/with-alpha', 'time/seconds']
 });
 
+StyleDictionaryPackage.registerTransformGroup({
+    name: 'products/email',
+    // TODO! update this according to the needs we have for the email templates
+    transforms: ['attribute/cti', 'name/cti/kebab', 'font-size/rem', 'size/px', 'color/css', 'color/with-alpha', 'time/seconds']
```

Comment:
@venusang do you need the `font-size` properties expressed in `rem` or in `px`?

Reply:
never mind, went ahead and changed to `px` (by the look of it seems [Foundation](https://get.foundation/emails/docs/typography.html#sass-reference) uses `px`)